packages/cache/src/CacheStore.spec.tsx

```diff
@@ -8,15 +8,26 @@ import { CacheStore } from './CacheStore'
 import { CacheStoreProvider } from './CacheStoreProvider'
 import { hashCacheKey } from './utils'
 
-const errorCache = (id: number) =>
-  cacheOptions({
-    cacheKey: ['key', id] as const,
-    cacheFn: () =>
-      sleep(ms('0.1s')).then(() =>
-        // eslint-disable-next-line @typescript-eslint/prefer-promise-reject-errors
-        Promise.reject(ERROR_MESSAGE)
-      ),
-  })
+const caches = {
```

Comment:
How about no depth

```tsx
cacheStore.remove(caches.errorCache(1))
```

 vs

```tsx
cacheStore.remove(errorCache(1))
```

Reply:
I think it's a good idea to separate the two functions, errorCache and successCache, and eliminate the depth. There seems to be no reason to bundle them into an object and add extra depth.